game/src/core/Game.java

```diff
@@ -778,6 +779,13 @@ private void onSetup() {
         createSystems();
         setupStage();
         userOnSetup.execute();
+
+        Server blocklyServer = new Server(hero);
+        try {
+            blocklyServer.start();
+        } catch (IOException e) {
+            throw new RuntimeException();
+        }
```

Comment:
Das Blocky-Dungeon müsste einen eigene Main bekommen und dort müsste dieser Teil in die `userOnSetup` Callback Funktion rein.

Ist also 0 Problemo

Reply:
DONE